plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -528,6 +556,7 @@ private String createContainer(String workspaceId,
               .withEntrypoint(toArrayIfNotNull(service.getEntrypoint()))
               .withLabels(service.getLabels())
               .withNetworkingConfig(networkingConfig)
+              .withUser(getUser())
```

Comment:
By the way I think the purpose of this PR is to solve permissions problems. Let's say that I'm logged in my Linux box using user `mario`. When starting a Che workspace I would expect it to run as user `mario` too. With permission to open files that I can read and only those. This user does not exist in the original recipe docker image but mounting the /etc/passwd will do the trick.